package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@acmucsd/membership-portal",
-  "version": "3.6.2",
+  "version": "3.6.3",
```

Comment:
I believe this should be a minor version update (3.7.0)–if you refer to [item 6 here](https://semver.org/#spec-item-6), patch versions are updated for bug fixes. Since you're technically changing the API interface, I think it warrants a bump.